lib/ui/semantics.dart

```diff
@@ -194,15 +195,20 @@ class SemanticsFlags {
   /// The focused element is usually the current receiver of keyboard inputs.
   static const SemanticsFlags isFocused = const SemanticsFlags._(_kIsFocusedIndex);
 
-  /// Whether the semantic node is currently disabled.
+  /// The semantics node has the quality of either being "enabled" or
+  /// "disabled".
+  ///
+  /// For example, a button can be enabled or disabled and therefore has enabled
+  /// state. Static text is usually either enabled nor disabled and does not
```

Comment:
either -> neither

Reply:
and does not -> and therefore does not
have enabled state -> have an "enabled" state